core/src/main/java/org/bitcoinj/base/internal/TimeUtils.java

```diff
@@ -70,6 +71,28 @@ public static Instant currentTime() {
         return Instant.now(clock);
     }
 
+    /**
+     * Returns the current time in "Bitcoin time", or a mocked out equivalent.
+     * Bitcoin time is used in the P2P protocol and represents seconds since epoch.
+     * It has no higher precision than the second.
+     */
+    public static Instant currentBitcoinTime() {
+        return currentTime().truncatedTo(ChronoUnit.SECONDS);
+    }
+
+    /**
+     * Ensures the given time is a "Bitcoin time".
+     * Bitcoin time is used in the P2P protocol and represents seconds since epoch.
+     * It has no higher precision than the second.
+     * @param time time to be checked for absence of precision higher than the second
+     * @return the time that was checked
+     */
+    public static Instant checkBitcoinTime(Instant time) {
+        Preconditions.checkArgument(time.getNano() == 0, () ->
+                "not a Bitcoin time: " + DateTimeFormatter.ISO_INSTANT.format(time));
+        return time;
+    }
+
```

Comment:
Do we want to "check and throw `RuntimeException`" or do we just want to truncate?